### DISCUSSION TITLE
Not a bug just wondering a question

### DISCUSSION THREAD

**Discussion by heqds** [TomSchimansky/CustomTkinter#1265] [General]
Am I able to print what a user enters into a text box? I am making a login system and am wondering how I can print what the user uses to login to help debug if they have a login error.
here is my code for reference
`import customtkinter

customtkinter.set_appearance_mode("dark")
customtkinter.set_default_color_theme("dark-blue")

root = customtkinter.CTk()
root.geometry("500x350")

frame = customtkinter.CTkFrame(master=root)
frame.pack(pady=20, padx=60, fill="both", expand=True)

label = customtkinter.CTkLabel(master=frame, text="Login System")
label.pack(pady=12, padx=10)

entry1 = customtkinter.CTkEntry(master=frame, placeholder_text="Username")
entry1.pack(pady=12, padx=10)

entry2 = customtkinter.CTkEntry(master=frame, placeholder_text="Password", show="*")
entry2.pack(pady=12, padx=10)

def login():
    print("Testing")

button = customtkinter.CTkButton(master=frame, text="Login", command=login)
button.pack(pady=12, padx=10)

checkbox = customtkinter.CTkCheckBox(master=frame, text="Remember me")
button.pack(pady=12, padx=10)

root.mainloop()`

**Comment by Wolf-SO** [comment]
For questions, it's better to go to [Discussions](https://github.com/TomSchimansky/CustomTkinter/discussions). (@TomSchimansky would you please move it?)

There is an indentation issue in the `login` function. BTW Code boxes can be made by triple backticks.

To get the value of an `CTkEntry` object, use the `get` method, see [wiki page about `CTkEntry`](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkEntry)

``````markdown
```python
customtkinter.set_appearance_mode("dark")
customtkinter.set_default_color_theme("dark-blue")

root = customtkinter.CTk()
root.geometry("500x350")

frame = customtkinter.CTkFrame(master=root)
frame.pack(pady=20, padx=60, fill="both", expand=True)

label = customtkinter.CTkLabel(master=frame, text="Login System")
label.pack(pady=12, padx=10)

entry1 = customtkinter.CTkEntry(master=frame, placeholder_text="Username")
entry1.pack(pady=12, padx=10)

entry2 = customtkinter.CTkEntry(master=frame, placeholder_text="Password", show="*")
entry2.pack(pady=12, padx=10)

def login():
    print("Testing")

button = customtkinter.CTkButton(master=frame, text="Login", command=login)
button.pack(pady=12, padx=10)

checkbox = customtkinter.CTkCheckBox(master=frame, text="Remember me")
button.pack(pady=12, padx=10)

root.mainloop()
```
``````
